funcr/funcr.go

```diff
@@ -100,6 +100,10 @@ type Options struct {
 	// details, see docs for Go's time.Layout.
 	TimestampFormat string
 
+	// LogInfoLevel tells func what key to use to log the integer level. By default,
+	// a "level" key will be used. If set it to "", the integer level will not be logged.
```

Comment:
nits:

`If not specified, the info level will be logged as "level".  If this is set to "", the info level will not be logged at all.`